sale_sourced_by_line/tests/test_sale_sourced_by_line.py

```diff
@@ -23,6 +23,25 @@ def setUp(self):
             {"name": "Test Warehouse", "code": "TWH"}
         )
 
+        # # TODO: Two tests failed because of a product not having a route set.
```

Comment:
There is no point on keeping commented code, does this come from the OCA?

Reply:
no, it is from braintec